arch/arm/src/armv6-m/arm_initialstate.c

```diff
@@ -74,7 +74,8 @@ void up_initial_state(struct tcb_s *tcb)
 
   /* Save the initial stack pointer */
 
-  xcp->regs[REG_SP]      = (uint32_t)tcb->adj_stack_ptr;
+  xcp->regs[REG_SP]      = (uint32_t)tcb->adj_stack_ptr +
```

Comment:
@xiaoxiang781216 - the old code's naming make sense and matched the semantic.  `xcp->regs[REG_SP] = (uint32_t)tcb->adj_stack_ptr`

In the new code you have changed the semantics and I feel very strongly that the naming has change.
tcb->adj_stack_ptr is NOW the stack base. Please call it that. `stackbase_ptr`

Then a the stack penetration test will start at the `stackbase_ptr` upward for the guard value.

Reply:
Ok, I will change to stack_base_ptr because it more similar with stack_alloc_ptr.

Reply:
Done.